src/rust/lib/graph-editor/src/component/node.rs

```diff
@@ -301,14 +257,15 @@ impl NodeModel {
         });
         display_object.add_child(&ports);
 
+        let frp = Frp{input};
 
-        let output_ports = OutputPortsEvents { shape_view_events:output_area.events.clone_ref() };
 
-        let frp = Frp{input,output_ports};
+        // TODO: Determine number of output ports based on node semantics.
+        let output_ports = OutputPorts::new(&scene, NonZeroU32::new(10).unwrap());
```

Comment:
I think instead of implicitly fixing it. it might be better to enable whoever will provide the value to do some error handling instead. That would allow us to, for example, show an error message at the source of the problem.